ant/build.gradle.kts

```diff
@@ -17,5 +18,5 @@ sourceSets {
 }
 
 runtimeJar {
-    manifest.attributes["Class-Path"] = "$compilerManifestClassPath kotlin-preloader.jar"
+    manifest.attributes["Class-Path"] = "$compilerManifestClassPath kotlin-preloader.jar kotlin-compiler.jar"
```

Comment:
This modification is used to support of setting propertiy: ```org.jetbrains.kotlin.cliMessageRenderer=FullPath```. This property is used in ```CLITool.kt```, line 189, to get the instance ```MessageRenderer.PLAIN_FULL_PATHS```, which is the same MessageRenderer as non-forking mode.

Reply:
It's nice that there actually is a way to select the message renderer! I didn't know that.

However, adding kotlin-compiler.jar to the classpath here is risky. At the very least, it can significantly slow down the loading time of the Ant task, since the compiler jar is very large. Let's just inline the constants from `MessageRenderer` instead, to avoid compile-time dependency on the compiler jar.